options.go

```diff
@@ -244,7 +243,7 @@ func (opts *options) handleFifos() (io.Writer, error) {
 	}
 
 	if generateFifoFilename || generateMetricFifoFilename {
-		dir, err := ioutil.TempDir(os.TempDir(), "fcfifo")
+		dir, err := os.MkdirTemp("", "fcfifo")
```

Comment:
These replacements were introduced in go1.16 and we claim to be able to build as far back as 1.14.

I think buildkite is currently on 1.15 so we would need to update that before merging this and remove go 1.14 and 1.15 from the GHA build matrix. Since firectl is a binary I think we can be a bit more aggressive in updating the go versions than the go sdk.

Reply:
Opened #95 to capture thoughts on this.